### ISSUE TITLE
Feedback to story on sandy beach erosion

### ISSUE THREAD

**Issue by Peter9192** [eucp-project/storyboards#64]
The story on sandy beach erosion received attention from users, and they've been kind enough to provide some comments. 

- It would be nice to show the skill of the application (comparing with observations)
- It would be helpful if the map was interactive / zoomable.
- For clarity, the "RCP8.5" should be renamed to "RCP8.5 - high end" or similar (to acknowledge increased SLR contribution from polar ice sheets). Alternatively and even better, the SLR projections could be based on the IPCC AR6.

Would it be possible to update the storyboard based on (some of) these comments, or can we provide an answer here? I'd be happy to help implement the changes.

**Comment by Peter9192** [comment]
> * It would be helpful if the map was interactive / zoomable.

If the figures' underlying data is available, we can try to make interactive figures. But perhaps a "high resolution" version of the image when you click on it would already be good enough?

**Comment by panosatha** [comment]
We would like to thank the users for their constructive comments and suggestions. We tried to update the storyboard where possible according to their comments:

- **It would be nice to show the skill of the application (comparing with observations)**

Unfortunetaly, it is not straightforward to validate the projections of SLR driven shoreline retreat due to lack of observations.
Nevertheless, we now changed the text in the 2nd tab to give more information on this:

_"Shoreline retreat is projected in the feature at sandy beach locations on 1 km alongshore grid around the European coastline using a simple equilibrium model  that can be computationally viable for the large spatial scale of this study.  Unfortunately, it is not possible to directly validate the model predictions,  since the observed SLR signal is not yet strong enough and the other processes involved in coastal erosion can mask the SLR effects."_

- **It would be helpful if the map was interactive / zoomable.**

We have now updated the maps with higher-resolution ones to allow zoom-in

- **For clarity, the "RCP8.5" should be renamed to "RCP8.5 - high end" or similar (to acknowledge increased SLR contribution from polar ice sheets). Alternatively and even better, the SLR projections could be based on the IPCC AR6.**

Unfortunately, updating the whole analysis with the new IPCC AR6 SLR projections will involve quite some work, which is out of the scope of the current study-case. Nevertheless, this would be a great idea for a follow-up study. To be more clear and according to the recommendation we have now changed all the text in the figures from "RCP 8.5" to "high-end RCP 8.5".
